pkg/virt-handler/device-manager/device_controller.go

```diff
@@ -74,7 +74,7 @@ func NewDeviceController(host string, maxDevices int, clusterConfig *virtconfig.
 	return controller
 }
 
-func (c *DeviceController) nodeHasDevice(devicePath string) bool {
+func (c *DeviceController) NodeHasDevice(devicePath string) bool {
```

Comment:
interesting... seems like nothing in the code is using this function, besides the unit test. I wonder, do we really need to make this public?

Reply:
I can move it to the node-labeller package, but I think it's more suitable when it's in the device manager package

Reply:
we can keep it here, I was asking why did it chance from nodeHasDevice to NodehasDevice